Adyen/Components/Base/ReadyToSubmitPaymentComponent.swift

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Foundation
+
+/// A payment component for which payment data is ready to be submitted.
+/// :nodoc:
+public final class ReadyToSubmitPaymentComponent: PaymentComponent {
```

Comment:
Not sure about the naming 🤔
What makes other Payment Components not "ready to Submit" ?

Reply:
deleted this one and used `EmptyPaymentComponent`, they are basically the same

Reply:
I am thinking about a better name. Maybe `InstantPaymentComponent ` ? 😆